core/platform_feature_list.py

```diff
@@ -62,8 +62,6 @@
     params.ParamNames.DUMMY_FEATURE_FLAG_FOR_E2E_TESTS,
     params.ParamNames.END_CHAPTER_CELEBRATION,
     params.ParamNames.CHECKPOINT_CELEBRATION,
-    params.ParamNames.ANDROID_BETA_LANDING_PAGE,
-    params.ParamNames.BLOG_PAGES,
```

Comment:
These should be moved to DEPRECATED_FEATURE_NAMES since they're now fully launched.

Reply:
Okay